operator/api/v1alpha1/flowtest_types.go

```diff
@@ -32,16 +32,20 @@ type FlowTestSpec struct {
 
 // FlowTestStatus defines the observed state of FlowTest
 type FlowTestStatus struct {
-	FailedMatch  flowv1beta1.Match  `json:"failedMatch"`
-	FailedFilter flowv1beta1.Filter `json:"failedFilter"`
-	// +kubebuilder:validation:Enum=Created;Running;Completed
+	// +nullable
+	PassedMatches []*flowv1beta1.Match `json:"passedMatches"`
+	// +nullable
+	PassedFilters []*flowv1beta1.Filter `json:"passedFilters"`
+	// +kubebuilder:default:="Created"
+	// +kubebuilder:validation:Enum=Created;Running;Completed;Error
```

Comment:
I will revert this back with the next PR. For now I wannted have the working alpha on the main branch